kinesis-produce-consume/pom.xml

```diff
@@ -0,0 +1,148 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<groupId>com.swl</groupId>
+	<artifactId>spring-cloud-stream-sample-kinesis</artifactId>
+	<version>0.0.1-SNAPSHOT</version>
+	<packaging>jar</packaging>
+
+	<name>KinesisProduceConsume</name>
+	<description>Demo project for Spring cloud stream kinesis binder</description>
+
+	<parent>
+		<groupId>org.springframework.boot</groupId>
+		<artifactId>spring-boot-starter-parent</artifactId>
```

Comment:
I think that's wrong.
Should be based on this parent:
```
<parent>
		<groupId>org.springframework.cloud</groupId>
		<artifactId>spring-cloud-stream-samples</artifactId>
		<version>1.2.0.BUILD-SNAPSHOT</version>
</parent>
```

Please, revise the structure of this pom like it is done in other samples near by.